prov/efa/src/efa_av.c

```diff
@@ -272,8 +272,13 @@ int efa_conn_rdm_init(struct efa_av *av, struct efa_conn *conn, bool insert_shm_
 	assert(!dlist_empty(&av->util_av.ep_list));
 	efa_rdm_ep = container_of(av->util_av.ep_list.next, struct efa_rdm_ep, base_ep.util_ep.av_entry);
 
-	peer = &conn->rdm_peer;
-	efa_rdm_peer_construct(peer, efa_rdm_ep, conn);
+	peer = (struct efa_rdm_peer *)calloc(1, sizeof(struct efa_rdm_peer));
```

Comment:
peer is a big object, should we use bufferpool to optimize the allocation?

Reply:
Tests are timing out now, so bufpool is not optional